arrow-libs/fx/arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/Fiber.kt

```diff
@@ -67,7 +67,7 @@ internal fun <A> Fiber(promise: UnsafePromise<A>, conn: SuspendConnection): Fibe
  */
 @Deprecated(
   "Use Deferred with KotlinX Coroutines Structured Concurrency",
-  ReplaceWith("async(ctx) { f() }", "kotlinx.coroutines.Deferred")
+  ReplaceWith("async(ctx) { f() }", "kotlinx.coroutines.async")
```

Comment:
Is `GlobalScope` needed here?
```suggestion
  ReplaceWith("GlobalScope.async(ctx) { f() }", "kotlinx.coroutines.async", "kotlinx.coroutines.GlobalScope")
```

Reply:
Actually I was going in the opposite direction 😂  `GlobalScope` shouldn't be used in general, instead people should use either the scopes provided by their target platform or build it themselves, so they can cancel whenever